### ISSUE TITLE
ERROR: dependency ‘cli’ is not available for package ‘testthat’

### ISSUE THREAD

**Issue by kimyen** [r-lib/testthat#695]
I am running a build that install `testthat`, but always failed for Ubuntu & R version 3.4:
```
Error : object ‘col_align’ is not exported by 'namespace:crayon'
ERROR: lazy loading failed for package ‘cli’
* removing ‘/usr/local/R/R-3.4.0/lib/R/library/cli’
ERROR: dependency ‘cli’ is not available for package ‘testthat’
* removing ‘/usr/local/R/R-3.4.0/lib/R/library/testthat’
```

**Comment by gaborcsardi** [comment]
You need to update the `crayon` package.

**Comment by kimyen** [comment]
Hello @gaborcsardi,

Thank you for a quick response! 
My package does not "directly" depends on `crayon` package. I would like to only add the direct dependencies to my build script if possible. Does `testthat` depends on a newer version of `crayon`?

**Comment by czeildi** [comment]
I had the same issue with crayon 1.3.2 but after updating it to 1.3.4 testing works fine. I believe the testthat package description file should contain 
crayon (>= 1.3.4) to force an update of crayon.

**Comment by jimhester** [comment]
Closed by #696
